main.js

```diff
@@ -84,6 +108,143 @@ app.on('before-quit', function (e) {
+        { label: "Redo", accelerator: "Shift+CmdOrCtrl+Z", selector: "redo:" },
+        { type: "separator" },
+        { label: "Cut", accelerator: "CmdOrCtrl+X", selector: "cut:" },
+        { label: "Copy", accelerator: "CmdOrCtrl+C", selector: "copy:" },
+        { label: "Paste", accelerator: "CmdOrCtrl+V", selector: "paste:" },
+        { label: "Select All", accelerator: "CmdOrCtrl+A", selector: "selectAll:" }
+      ]
+    },
+    {
+      label: 'Edit',
+      submenu: [
+        { label: "Undo", accelerator: "CmdOrCtrl+Z", selector: "undo:" },
+        { label: "Redo", accelerator: "Shift+CmdOrCtrl+Z", selector: "redo:" },
+        { type: "separator" },
+        { label: "Cut", accelerator: "CmdOrCtrl+X", selector: "cut:" },
+        { label: "Copy", accelerator: "CmdOrCtrl+C", selector: "copy:" },
+        { label: "Paste", accelerator: "CmdOrCtrl+V", selector: "paste:" },
+        { label: "Select All", accelerator: "CmdOrCtrl+A", selector: "selectAll:" }
+      ]
+    },
+    {
+      label: 'View',
+      submenu: [
+        {
+          label: 'Reload',
+          accelerator: 'CmdOrCtrl+R',
+          click: function(item, focusedWindow) {
+            if (focusedWindow) {
+              focusedWindow.reload();
+            }
+          }
+        },
+        {
+          label: 'Toggle Developer Tools',
+          accelerator: (function() {
+            if (platform == 'mac') {
+              return 'Alt+Command+I';
+            } else {
+              return 'Ctrl+Shift+I';
+            }
+          })(),
+          click: function(item, focusedWindow) {
+            if (focusedWindow) {
+              focusedWindow.toggleDevTools();
+            }
+          }
+        },
+      ]
+    },
+    {
+    label: 'Window',
+    submenu: [
+      {
+        label: 'Minimize',
+        accelerator: 'Command+M',
+        selector: 'performMiniaturize:'
+      },
+      {
+        label: 'Close',
+        accelerator: 'Command+W',
+        selector: 'performClose:'
+      },
+      {
+        type: 'separator'
+      },
+      {
+        label: 'Bring All to Front',
+        selector: 'arrangeInFront:'
+      }
+    ]
+  }
+  ]);
+  menu.setApplicationMenu(appMenu);
+
+  trayMenu = new tray(__dirname + '/imgs/menubar_icon.png');
+  var contextMenu = menu.buildFromTemplate([
+    {
+      label: 'Start Local Server', type: 'normal', click: function () {
+      start_local_server();
+    }
+    },
+    {
+      label: 'Shutdown Local Server', type: 'normal', click: function () {
+      request('http://localhost:18469/api/v1/shutdown', function (error, response, body) {
+        if (!error && response.statusCode == 200) {
+          console.log('Shutting down server');
+        } else {
+          console.log('Server does not seem to be running.');
+        }
+      });
+    }
+    },
+    {type: 'separator'},
+    {label: 'View Debug Log', type: 'normal', click: function() {
+      // Open Debug Log Wherever It Is
+
```

Comment:
I suppose this line needs to be more than a comment to make opening the debug logs work. ;)